README.md

```diff
@@ -41,6 +41,7 @@ This README provides detailed information on how to set up, develop, and deploy
     - [Milvus](#milvus)
     - [Qdrant](#qdrant)
     - [Redis](#redis)
+    - [pgvector](#pgvector)
```

Comment:
```suggestion
    - [Postgres (pgvector)](#pgvector)
```

I'd recommend using "Postgres" in the title since it's more familiar with developers